src/components/Views/ObservationEditor/index.js

```diff
@@ -72,7 +78,9 @@ function mapDispatchToProps(dispatch: Dispatch): DispatchProps {
           routeName: 'MapView'
         })
       ),
-    showSavedModal: () => dispatch(modalShow('saved'))
+    showSavedModal: () => dispatch(modalShow('saved')),
```

Comment:
I think you might need to fix ObservationEditor tests to include these props

Reply:
they seem okay!